.pullapprove.yml

```diff
@@ -1,8 +1,10 @@
 approve_by_comment: true
-approve_regex: ^LGTM
+approve_regex: '^(lgtm|LGTM)'
```

Comment:
On Tue, Jul 26, 2016 at 08:23:37PM -0700, Qiang Huang wrote:
> > @@ -1,8 +1,10 @@
> >  approve_by_comment: true
> > -approve_regex: ^LGTM
> > +approve_regex: '^(lgtm|LGTM)'
>
> I know non-maintainers' lgtm won't count, just don't think it
> necessary to introduce such flexibility, my 2 cents, I don't object
> if other people want this.

“The shift button is not that hard to push” is reasoning I can get
behind ;).  But yeah, whatever maintainers want (the image-spec
maintainers like a lot of flexibility, opencontainers/image-spec#108
;).